config/scout.php

```diff
@@ -139,12 +142,41 @@
         'host' => env('MEILISEARCH_HOST', 'http://localhost:7700'),
         'key' => env('MEILISEARCH_KEY'),
         'index-settings' => [
+            Comment::class => [
+                'filterableAttributes' => [
+                    'ArticleID',
+                    'ArticleType',
+                    'commentable_id',
+                    'commentable_type',
+                    'created_at',
+                    'user_id',
+                ],
+                'sortableAttributes' => ['created_at'],
+                'searchableAttributes' => ['body'],
+            ],
+
+            ForumTopicComment::class => [
+                'filterableAttributes' => [
+                    'forum_topic_id',
+                    'author_id',
+                    'created_at',
+                ],
+                'sortableAttributes' => ['created_at'],
+                'searchableAttributes' => ['body'],
```

Comment:
Consider putting `searchableAttributes` before `sortableAttributes` for consistency. (here and Comment)

Reply:
Updated in latest.